tests/test_blocks.py

```diff
@@ -245,7 +245,23 @@ def test_parent_class(self):
     def test_render(self):
         """Test PullQuoteBlock.render renders as expected."""
         block = internal_blocks.PullQuoteBlock()
-        expected = '<blockquote class="pquote">some text</blockquote>'
+        expected = '<blockquote class="pquote">some text</blockquote>\n'
```

Comment:
is this extra newline likely to break any of existing usage of this block?

Reply:
No. The only reason it exists there is because of my IDE settings that append newline by default. (And i like this rule)